src/twisted/spread/test/test_pbfailure.py

```diff
@@ -390,7 +405,10 @@ def generatorFunc():
         self.assertRaises(StopIteration, copy.throwExceptionIntoGenerator, gen)
         self.assertEqual(len(exception), 1)
         exc = exception[0]
-        self.assertEqual(exc.remoteType, "exceptions.AttributeError")
+        if _PY3:
```

Comment:
As I keep saying, `if _PY3` is an anti-pattern and we should do it in as few places as possible.  It's just a recipe for coverage problems in the future.  We definitely shouldn't be doing it in test bodies, since it is an instance of [the "conditional test logic" code smell](http://xunitpatterns.com/Conditional%20Test%20Logic.html).

A better way to express this here would be `expectedExceptionType = qual(AttributeError)`.

(That said, this is not bad enough to block on, just please keep it in mind, since every `if _PY3` is technical debt we're eventually going to need to get rid of.)